File: VeriBank.Business.Transactions/MoneyTransfer/MoneyTransferTransaction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using VeriBank.Business.Transactions.Infrastructure;
using VeriBank.Business.Transactions.Infrastructure.BaseClasses;
using VeriBank.Business.Transactions.Infrastructure.Interfaces;
using VeriBank.Business.Transactions.Managers.MoneyTransfer;

namespace VeriBank.Business.Transactions.MoneyTransfer
{
    public class MoneyTransferTransaction : TransactionBase<MoneyTransferManager>
    {
        public override void Execute(ITransactionRequest request, ITransactionResponse response)
        {
            Manager.DoAction(request, response);
        }

        public override void Fetch(ITransactionRequest request, ITransactionResponse response)
        {

        }

        public override void Confirm(ITransactionRequest request, ITransactionResponse response)
        {

        }
    }
}

File: VeriBank.Web/Scripts/Transactions/MoneyTransferTransaction.ts
namespace VeriBank.Business.Transactions.MoneyTransfer {
    interface ITransaction {
        Start: (request: cm.RequestMessage, response: cm.ResponseMessage) => void;
        Fetch: (request: cm.RequestMessage, response: cm.ResponseMessage) => void;
        Confirm: (request: cm.RequestMessage, response: cm.ResponseMessage) => void;
        Execute: (request: cm.RequestMessage, response: cm.ResponseMessage) => void;
    }

    export class MoneyTransferTransaction implements ITransaction {
        Start(request: cm.RequestMessage, response: cm.ResponseMessage) {

        }
        Fetch(request: cm.RequestMessage, response: cm.ResponseMessage) {

        }
        Confirm(request: cm.RequestMessage, response: cm.ResponseMessage) {

        }
        Execute(request: cm.RequestMessage, response: cm.ResponseMessage) {

        }
    }
}